lib/process-services/src/lib/task-list/components/task-details.component.ts

```diff
@@ -395,6 +396,24 @@ export class TaskDetailsComponent implements OnInit, OnChanges, OnDestroy {
         this.loadDetails(taskId);
     }
 
+    onTaskClaim(taskClaimStatus: TaskClaimModel): void {
```

Comment:
don't get it, why do we need to have 1 event instead of 2 and then having an if?

Reply:
Updated